crates/core/src/yarn_fn/function_wrapping.rs

```diff
@@ -37,6 +39,10 @@ use variadics_please::all_tuples;
 pub trait YarnFn<Marker>: Clone + Send + Sync {
```

Comment:
The documentation of this trait is outdated now. See the [`YarnCommand`](https://docs.rs/bevy_yarnspinner/latest/bevy_yarnspinner/prelude/trait.YarnCommand.html) method for how to rewrite this.

Reply:
I didn't exactly copy the Command docs because here we're taking a full system so it's more like "do whatever bevy wants plus a few restrictions on the input parameters`